### PR TITLE
{data}[foss/2022b, GCCcore/12.2.0] GDAL v3.6.2 w/ Python 3.10.8, libgeotiff v1.7.1

### PR DESCRIPTION
**Pull request by jfgrimm**
requires:

- [x] #16912 
- [x] #17437  
- [x] #17190
- [x] #17191
- [x] #17192
- [x] #17193
- [x] #17196 
- [x] #17107

(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    ('libxml2', '2.10.3'),
+    ('libpng', '1.6.38'),
+    ('libjpeg-turbo', '2.1.4'),
+    ('JasPer', '4.0.0'),
+    ('LibTIFF', '4.4.0'),
+    ('zlib', '1.2.12'),
+    ('cURL', '7.86.0'),
+    ('PCRE', '8.45'),
+    ('PROJ', '9.1.1'),
+    ('libgeotiff', '1.7.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb:44]: @jfgrimm PR is missing for `libgeotiff` easyconfig?

File: easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    ('SciPy-bundle', '2023.01'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb:45]: Should be bumped to `2023.02`, see #16912

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb:44]: added to this PR

File: easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb (hunk 2 above)
Comment (jfgrimm) [inline easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb:45]:
```suggestion
    ('SciPy-bundle', '2023.02'),
```

Comment (jfgrimm) [reply easybuild/easyconfigs/g/GDAL/GDAL-3.6.2-foss-2022b.eb]: done

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3104.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/56e6852e9b43576d9cb8b159c4000a70 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jfgrimm!

**Pull request merged**